components/supervision_dispatch/examples/late_component_launcher.cpp

```
@@ -105,12 +108,29 @@ namespace {
     // Task 4a: spawn_worker()/worker_slot/task scaffolding.
     // ========================================================================
 
+    // Which worker binary spawn_worker() launches: the graceful
+    // late_component_worker, the mid-epoch-departing
+    // late_component_failing_worker, or the late_component_aborting_worker that
+    // models a true, unmodeled crash (std::exit(-1), no teardown at all).
+    enum class worker_variant : std::uint8_t
+    {
+        normal = 0,
+        //failing = 1,
+        //aborting = 2
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If HPX_REGISTER_ACTION_DECLARATION is a macro then please configure it.

(unknownMacro)